### PR TITLE
Uniform the code used to execute cmsRun and scriptExe

### PR DESCRIPTION
**Pull request by mmascher**

### CODE REVIEW

**Comment by ericvaandering** [comment]
retest this please

**Comment by ericvaandering** [comment]
retest this please

**Comment by ericvaandering** [comment]
retest this please

**Comment by ericvaandering** [comment]
retest this please

**Comment by ericvaandering** [comment]
retest this please

**Comment by ericvaandering** [comment]
retest this please

**Comment by ericvaandering** [comment]
retest this please

File: scripts/CMSRunAnalysis.py [hunk 1]
```diff
@@ -297,45 +296,55 @@ def stopDashboardMonitoring(myad):
...
         logCMSSWSaved = True
         open('logCMSSWSaved.txt', 'a').close()
         os.utime('logCMSSWSaved.txt', None)
         return
-    print "======== CMSSW OUTPUT STARTING ========"
 
-    outfile = "cmsRun-stdout.log"
-
-    # check size of outfile
+    outfiles = [ ("TewakPSet-stdout.log", "TWEAKPSET"), ( "cmsRun-stdout.log", "CMSSW")]
```
Comment (juztas) [inline scripts/CMSRunAnalysis.py]: typo? '"TewakPSet-stdout.log"'

**Comment by ericvaandering** [comment]
retest this please

**Comment by ericvaandering** [comment]
retest this please

**Comment by ericvaandering** [comment]
test this please

**Comment by ericvaandering** [comment]
test this please

**Comment by ericvaandering** [comment]
test this please

**Comment by ericvaandering** [comment]
test this please

**Comment by ericvaandering** [comment]
test this please

**Comment by belforte** [comment]
@mmascher I noticed this  by chance, I thought the work in the subject was done, long ago indeed. Is this still an open issue ?

**Comment by belforte** [comment]
@mmascher shall we close this ? I think there's no chance you will get back to it, unless you did this already.

**Comment by belforte** [comment]
there's really no way that we get to review such basic things unless there's a dire need

**Pull request closed without merging**